lib/school_house_web/templates/layout/_lesson_menu.html.leex

```diff
@@ -2,287 +2,343 @@
-    </li>
-    <li>
-        <%= lesson_link @conn, :ecto, :querying_basics do %>
-        4. <%= gettext("Querying: Basics") %>
-        <% end %>
-    </li>
-    <li>
-        <%= lesson_link @conn, :ecto, :querying_advanced do %>
-        5. <%= gettext("Querying: Advanced") %>
-        <% end %>
-    </li>
-    <h3 class="font-bold text-xl text-brand-gray text-bold mb-2 pt-6"><%= gettext("Storage") %></h3>
-    <li>
-        <%= lesson_link @conn, :storage, :ets do %>
-        1. Erlang Term Storage (ETS)
-        <% end %>
-    </li>
-    <li>
-        <%= lesson_link @conn, :storage, :mnesia do %>
-        2. Mnesia
-        <% end %>
-    </li>
-    <li>
-        <%= lesson_link @conn, :storage, :cachex do %>
-        3. Cachex <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-    </li>
-    <li>
-        <%= lesson_link @conn, :storage, :redix do %>
-        4. Redix <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-    </li>
-    </ul>
-    <ul class="px-4 w-full sm:w-1/4 lg:w-1/5 border-gray-600 pb-6 pt-6 lg:pt-3">
-        <h3 class="font-bold text-xl text-brand-gray text-bold mb-2"><%= gettext("Miscellaneous") %></h3>
-        <li>
-        <%= lesson_link @conn, :misc, :plug do %>
-            1. Plug <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :eex do %>
-            2. Embedded Elixir (EEx)
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :debugging do %>
-            3. Debugging
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :nerves do %>
-            4. Nerves <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :guardian do %>
-            5. Guardian <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :poolboy do %>
-            6. Poolboy <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :distillery do %>
-            7. Distillery <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-        </li>
-        <li>
-        <%= lesson_link @conn, :misc, :nimble_publisher do %>
-            8. NimblePublisher <span class="text-xs italic"><%= gettext("library") %></span>
-        <% end %>
-        </li>
-    </ul>
+        <!-- First column -->
+        <ul class="px-4 w-full sm:w-1/4 lg:w-1/5 border-gray-600 border-b sm:border-r lg:border-b-0 pb-6 pt-6 lg:pt-3">
+            <!-- Basics -->
+            <li>
+                <p class="font-bold text-xl text-brand-gray text-bold mb-2"><%= gettext("Basics") %></p>
+                <ol class="list-decimal ml-6">
```

Comment:
TIL `list-decimal`, I missed this in their documentation :+1:

Reply:
Yes, Tailwind lists are unstyled by default. https://tailwindcss.com/docs/preflight#lists-are-unstyled